scripts/backup

```diff
@@ -33,6 +33,12 @@ ynh_backup --src_path="/etc/nginx/conf.d/$domain.d/$app.conf"
 
 ynh_backup --src_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
 
+#=================================================
+# BACKUP LOGS
+#=================================================
+
+ynh_backup --src_path="/var/log/nginx/$domain-access.log"
```

Comment:
@OniriCorpe : uuuuh wut x_x I'm not sure to understand why we would backup/restore the nginx log of the entire domain x_X